modules/nw-bgp-about.adoc

```diff
@@ -0,0 +1,55 @@
+// Module included in the following assemblies:
+//
+// * networking/bgp_routing/about-bgp-routing.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="nw-bgp-about_{context}"]
+= About Border Gateway Protocol (BGP) routing
+
+With route advertisements enabled, {product-title} Networking supports the following features:
+
+- Importing routes from the provider network
+- Exporting routes into the provider network
+- Supporting multi-homing, link redundancy, and fast convergence for provider network gateways
+////
+https://github.com/openshift/openshift-docs/pull/85842/files#r1900917285
+- Improving throughput and reducing packet overhead
+////
```

Comment:
I would remove this since it is mentioning the route advertisements feature which is not in scope of this PR.